### PR TITLE
fix incorrect currency symbol

### PR DESCRIPTION
**Pull request by ludrob**
### Description
We get an incorrect symbol on the payment term because we compute some price client side and forgot to pass in the symbol.
It turns out that we already compute the prices server side, no need to do this in the client.

related PR: https://github.com/Financial-Times/next-subscribe/pull/2537

### Ticket
https://financialtimes.atlassian.net/browse/ACQ-2186

### CODE REVIEW

File: utils/payment-term.js [hunk 1]
```diff
@@ -138,13 +156,10 @@ class PaymentTerm {
...
-					? this.getMontlyPriceFromPeriod(
-							update.amount,
-							update.currency,
-							update.value
-					  )
-					: update.monthlyPrice;
+				const updatedPrice = this.getUpdatedPrice(update);
+				if (updatedPrice) {
+					monthlyPrice.innerHTML = updatedPrice;
+				}
```
Comment (gonzalogcontacto) [inline utils/payment-term.js:162]: We are calculating the monthly price on `next-subscribe` 👉 https://github.com/Financial-Times/next-subscribe/blob/80da6fe05f33a79b3000faee396ee384822de107/server/helpers/payment-terms.js#L130, so always the monthly price is provided?

I realise at some offers like print, we are preparing a wrong monthly price ("0"), thats why i think we are here repeating the operation here (including if the monthly price provided is good calculated, like on trial and others)

Its possible to get one only source of true? Maybe im missing something?

Comment (gonzalogcontacto) [reply utils/payment-term.js:162]: We're applying the calculation for all offers we get from membership 👉 https://github.com/Financial-Times/next-subscribe/blob/80da6fe05f33a79b3000faee396ee384822de107/server/helpers/payment-terms.js#L105

Comment (ludrob) [inline utils/payment-term.js:162]:
```suggestion
				if (updatedPrice) {
					monthlyPrice.innerHTML = update.monthlyPrice;
				}
```

**Review by gonzalogcontacto — APPROVED** [review]
Less duplications is good to avoid repeated issues 🙏🏻  great team work here! 🙌

File: utils/payment-term.js [hunk 2]
```diff
@@ -137,14 +131,8 @@ class PaymentTerm {
...
 			}
-			if (monthlyPrice) {
-				monthlyPrice.innerHTML = this.isValidPeriod(update.value)
-					? this.getMontlyPriceFromPeriod(
-							update.amount,
-							update.currency,
-							update.value
-					  )
-					: update.monthlyPrice;
+			if (monthlyPrice && update && update.monthlyPrice) {
```
Comment (JuanTincho) [inline utils/payment-term.js:134]: In line 120 we cover if Update doesn't exist: 
```suggestion
			if (monthlyPrice &&  update.monthlyPrice) {
```

File: utils/payment-term.js [hunk 3]
```diff
@@ -137,14 +131,8 @@ class PaymentTerm {
...
 			}
-			if (monthlyPrice) {
-				monthlyPrice.innerHTML = this.isValidPeriod(update.value)
-					? this.getMontlyPriceFromPeriod(
-							update.amount,
-							update.currency,
-							update.value
-					  )
-					: update.monthlyPrice;
+			if (monthlyPrice &&  update.monthlyPrice) {
```
Comment (ludrob) [inline utils/payment-term.js:134]:
```suggestion
			if (monthlyPrice && update.monthlyPrice) {
```

**Review by JuanTincho — APPROVED** [review]

File: utils/payment-term.js [hunk 4]
```diff
@@ -137,14 +131,8 @@ class PaymentTerm {
 			if (trialPrice) {
```
Comment (joejeet) [inline utils/payment-term.js:131]: Question: do we need to check here for `update.trialPrice and update.price` as well before assigning it to the inner HTML of the elements?

Comment (ludrob) [reply utils/payment-term.js:131]: sorry, I've just seen your comment after I've merged.
We should always have a trial price if the offer has trial terms, so I'm hoping that's fine as it is.

**Pull request merged**